src/main/java/com/google/devtools/build/lib/skylarkbuildapi/ProtoConfigurationApi.java

```diff
@@ -23,7 +25,26 @@
 @SkylarkModule(
     name = "proto",
     category = SkylarkModuleCategory.CONFIGURATION_FRAGMENT,
-    doc = "A configuration fragment representing protocol buffers."
+    doc = "A configuration fragment representing protocol buffers. "
```

Comment:
Hm, it's kind of dangerous to do this; documentation or not, people will start depending on this. A scary name (`protoc_opts_DO_NOT_USE_OUR_I_ WILL_BREAK_YOU_WITHOUT_MERCY`) helps. A bit. Unfortunately, I don't have access to the document

https://docs.google.com/document/d/1u95vlQ1lWeQNR4bUw5T4cMeHTGJla2_e1dHHx7v4Dvg/edit?usp=sharing

I suppose it's elucidated there what your migration plan is?

Reply:
Yeah, the document outlines a migration plan and what's necessary to make that happen. I'll add it to the proposals repo when it's ready for review.

I agree that it's dangerous and that a scarier name would be prefarable. Alternatively, we could add a flag for whitelisting packages that can use `protoc_opts` and `strict_deps`, but, AAICT, `@SkylarkConfigurationField` doesn't expose the necessary information yet, so that won't work for `protoc`.

Btw, what's the accepted convention for changes that aren't ready for review? I used a draft PR to indicate that. Is it better to add `NOT READY FOR REVIEW` to the title or description?

Reply:
Well, all I know is that @jin added me as a reviewer so I dutifully started reviewing the change :) I'm not sure why he did that, but I guess a big scary "NOT READY FOR REVIEW" phrase would have helped.

I'll refrain from formulating an opinion until I see what's in the document and until you explicitly ask me to do so.

Reply:
My bad, sorry. I added @lberki prematurely to this and https://github.com/bazelbuild/bazel/pull/9002. The convention of using draft PRs is correct.

Reply:
No problem! We'll revisit this when it's time :)